includes/admin/class-wc-stripe-upe-compatibility-controller.php

```diff
@@ -63,8 +63,8 @@ public function add_compatibility_notice() {
 				],
 				[
 					'name'         => 'WooCommerce',
-					'version'      => $this->get_wc_version(),
-					'is_supported' => version_compare( $this->get_wc_version(), WC_Stripe_UPE_Compatibility::MIN_WC_VERSION, '>=' ),
+					'version'      => $this->get_wc_version() ? $this->get_wc_version() : '[version-not-found]',
+					'is_supported' => defined( 'WC_VERSION' ) && version_compare( $this->get_wc_version(), WC_Stripe_UPE_Compatibility::MIN_WC_VERSION, '>=' ),
```

Comment:
Added a check here too, why not 🤷 
I _believe_ that `WC_VERSION` is always defined, but who knows at this point 🤷

Reply:
+1 on `WC_VERSION` should be defined in [WooCommerce core](https://github.com/woocommerce/woocommerce/blob/trunk/includes/class-woocommerce.php#L260). Without this constant, I imagine WooCommerce is not loaded.

Instead of doing `defined( 'WC_VERSION' )` everywhere, 1 other option is to have 1 plugin check at the main `woocommerce-gateway-stripe.php` file. Throw an error and say that the plugin can't be loaded because the constant is missing.

Reply:
Fun fact, there is also `WOOCOMMERCE_VERSION`.

Reply:
That's actually a good point - we already have a check for WC here: https://github.com/woocommerce/woocommerce-gateway-stripe/blob/3f1a259f634d9499178e4d9dd6f7a2681b391873/woocommerce-gateway-stripe.php#L621-L624

So we should be covered 👍 
Removed the unnecessary checks in 6aec9af8